x/concentrated-liquidity/bench_test.go

```diff
@@ -0,0 +1,104 @@
+package concentrated_liquidity_test
+
+import (
+	"fmt"
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/simapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/tendermint/tendermint/crypto/secp256k1"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	"github.com/osmosis-labs/osmosis/v14/app"
+	"github.com/osmosis-labs/osmosis/v14/app/apptesting"
+	sdkrand "github.com/osmosis-labs/osmosis/v14/simulation/simtypes/random"
+	clmodel "github.com/osmosis-labs/osmosis/v14/x/concentrated-liquidity/model"
+)
+
+// benchmarkCrossTick performs bench test on the amount of computational resources it takes to cross ticks N times.
+// We achieve the goal of doing so initializing N amount of positions each with 1 * denom0, 1 * denom1.
+// This way performing a swap of N tokens would cause us to cross N amount of ticks, being able to bench `crossTick`.
+func benchmarkCrossTick(numCrossTick int, b *testing.B) {
+	b.StopTimer()
+
+	blockStartTime := time.Now().UTC()
```

Comment:
let's not use `time.Now()`. Instead, set it to some deterministic value please

Reply:
Changed!